tfhe/benches/integer/signed_bench.rs

```diff
@@ -651,111 +814,248 @@ macro_rules! define_server_key_bench_binary_scalar_clean_inputs_fn (
+    method_name: scalar_left_shift_parallelized,
+    display_name: left_shift,
+    rng_func: shift_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_right_shift_parallelized,
+    display_name: right_shift,
+    rng_func: shift_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_rotate_left_parallelized,
+    display_name: rotate_left,
+    rng_func: shift_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_rotate_right_parallelized,
+    display_name: rotate_right,
+    rng_func: shift_scalar
+);
+
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_max_parallelized,
+    display_name: max,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_min_parallelized,
+    display_name: min,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_eq_parallelized,
+    display_name: equal,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_ne_parallelized,
+    display_name: not_equal,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_lt_parallelized,
+    display_name: less_than,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_le_parallelized,
+    display_name: less_or_equal,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_gt_parallelized,
+    display_name: greater_than,
+    rng_func: default_scalar
+);
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: scalar_ge_parallelized,
+    display_name: greater_or_equal,
+    rng_func: default_scalar
+);
+
+criterion_group!(
+    default_scalar_parallelized_ops,
+    scalar_add_parallelized,
+    scalar_sub_parallelized,
+    scalar_mul_parallelized,
+    signed_scalar_div_parallelized,
+    signed_scalar_rem_parallelized, // For scalar rem == div_rem
+    // signed_scalar_div_rem_parallelized,
+    scalar_bitand_parallelized,
+    scalar_bitor_parallelized,
+    scalar_bitxor_parallelized,
+    scalar_left_shift_parallelized,
+    scalar_right_shift_parallelized,
+    scalar_rotate_left_parallelized,
+    scalar_rotate_right_parallelized,
+);
+
+criterion_group!(
+    default_scalar_parallelized_ops_comp,
+    scalar_max_parallelized,
+    scalar_min_parallelized,
+    scalar_eq_parallelized,
+    scalar_ne_parallelized,
+    scalar_lt_parallelized,
+    scalar_le_parallelized,
+    scalar_gt_parallelized,
+    scalar_ge_parallelized,
+);
+
 define_server_key_bench_binary_scalar_clean_inputs_fn!(
     method_name: unchecked_scalar_left_shift_parallelized,
-    display_name: scalar_left_shift,
```

Comment:
this matches the unsigned benches ?

Reply:
Yes `scalar_` should never appear in the display name.